lib/run.tsx

```diff
@@ -74,24 +74,26 @@ const run = async (string: string, _opts: RunOpts = {}) => {
     } as RunOptions) as Promise<RMDXModule>;
   };
 
-  const { Toc: _Toc, toc, default: Content, ...exports } = await exec(string);
+  const { Toc: _Toc, toc, default: Content, stylesheets, ...exports } = await exec(string);
 
   const tocMdx = tocToMdx(toc, components);
-  const { default: Toc } = await exec(compile(tocMdx), { useMDXComponents: () => ({ p: Fragment }) });
+  const { default: Toc } = await exec(await compile(tocMdx), { useMDXComponents: () => ({ p: Fragment }) });
 
   return {
     default: () => (
       <Contexts terms={terms} baseUrl={baseUrl} variables={variables}>
+        {!!stylesheets && stylesheets.map(stylesheet => <style>{stylesheet}</style>)}
```

Comment:
It looks like we're adding the `<style>` wherever we run rdmd right? So the markup will be something like 
```
<div class="markdown-body">
  <style>
     ...
  </style>
  ...
</div>
```
It's worth noting that putting a `style` tag outside of the `<head>` is _technically_ not valid html. Seems like it'll work fine though? We just might want to be mindful that
1. This doesn't create any FOUC in the readme app once it's all wired up
2. Any of our customers won't have a problem that their hub isn't going to pass strict HTML validation. Not sure if this would impact seo or a11y compliance. Probably not?

Found a [whatwg thread all about this](https://github.com/whatwg/html/issues/1605). You used to be able to use `<style scoped>` on some browsers for this, but it was deprecated.

Reply:
Yea, that's a good point. I'd like to keep it to prevent a FOUC, but maybe I can have it portal into the head async.

IIRC, when rendering server side, we render this output and inject the html into the doc. So this component won't have access to the head until it's fully rehydrated in the client.

Reply:
Yeah true -- & having to wait till the app hydrates to tack on the stylesheet would have waaaaay more of a FOUC than what you're currently doing.

Reply:
I think for this first pass it’s fine to inject these `<style>` tags outside of the head; we can always revisit down the line if we notice issues/get complaints!